packages/core/strapi/lib/commands/opt-out-telemetry.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const { resolve } = require('path');
+const fse = require('fs-extra');
+const chalk = require('chalk');
+const fetch = require('node-fetch');
+const machineID = require('../utils/machine-id');
+
+const readPackageJSON = async path => {
+  try {
+    const packageObj = await fse.readJson(path);
+    const uuid = packageObj.strapi.uuid;
```

Comment:
You will have to use lodash.get this could be break if the user decided to remove the strapi key

Reply:
I have implemented the check for `uuid` in the main function so I would not need to bring in additional packages. The process will be terminated if `uuid` will be undefined since that automatically means user already opted out. Is there a flow in this logic and I would still need to use lodash?

Reply:
Hum I don't see where you check if packageJson.strapi exists.

You also can have an error on L55 with `packageObj.strapi.telemetryDisabled` 

if the strapi key is not in the package.json all of this will breakfrom what I'm seeing. have you tried it?

Reply:
Here is the exact error you get 

```
Error: Cannot read property 'uuid' of undefined
TypeError: (intermediate value) is not iterable
    at optOutTelemetry (/Users/alex/dev/strapi/strapi-v4/packages/core/strapi/lib/commands/opt-out-telemetry.js:53:30)
```

Not very friendly :)